.github/workflows/release.yml

```diff
@@ -865,13 +858,19 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        network: [local]
         arch: [amd64]
         node: [collator-node-local, instant-seal-node]
         include:
-          - network: local
+          - node: collator-node-local
```

Comment:
I'd be OK with not publishing this one at all now that I look at it. Fine to do so, but also fine to have less in the release.

Reply:
Be nice if we could remove it from the build entirely if we don't use it elsewhere